From 6adf534de5daa77e5a1220b659dc8032cead826e Mon Sep 17 00:00:00 2001
From: Martin Stransky <stransky@redhat.com>
Date: Mon, 22 Oct 2018 09:14:07 +0000
Subject: [PATCH] Bug 1500850 - [Wayland] Add missing dbus header
 dbus-glib-lowlevel.h. r=jhorak, a=RyanVM

Differential Revision: https://phabricator.services.mozilla.com/D9365

--HG--
extra : source : 5203edd6ea29e4b73681674d2a5268dd6a310102
extra : intermediate-source : c7d13f90ff314656131df828dc1c792b55cc379a
---
 widget/xremoteclient/DBusRemoteClient.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/widget/xremoteclient/DBusRemoteClient.cpp b/widget/xremoteclient/DBusRemoteClient.cpp
index ba5bda0..b2f03dc 100644
--- a/widget/xremoteclient/DBusRemoteClient.cpp
+++ b/widget/xremoteclient/DBusRemoteClient.cpp
@@ -13,6 +13,7 @@
 #include "nsPrintfCString.h"
 
 #include <dlfcn.h>
+#include <dbus/dbus-glib-lowlevel.h>
 
 using mozilla::LogLevel;
 static mozilla::LazyLogModule sRemoteLm("DBusRemoteClient");
-- 
2.7.4

